From c4f0cbc414b2ef3750ac20fcad0e286f7d4c33a6 Mon Sep 17 00:00:00 2001
From: "yildirim340458@gmail.com" <yildirim340458@gmail.com>
Date: Mon, 8 Dec 2025 23:04:02 +0300
Subject: [PATCH 10/23] Changeset: 8

---
 Assets/Scenes/LWL2.unity | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Assets/Scenes/LWL2.unity b/Assets/Scenes/LWL2.unity
index fb0473f6..e9c0d941 100644
--- a/Assets/Scenes/LWL2.unity
+++ b/Assets/Scenes/LWL2.unity
@@ -606,7 +606,7 @@ Transform:
   m_GameObject: {fileID: 572292125}
   serializedVersion: 2
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: -19.2, y: 4.53, z: 0}
+  m_LocalPosition: {x: -19.2, y: 4.67, z: 0}
   m_LocalScale: {x: 0.763, y: 0.6840596, z: 1}
   m_ConstrainProportionsScale: 0
   m_Children: []
@@ -788,7 +788,7 @@ Transform:
   m_GameObject: {fileID: 686194285}
   serializedVersion: 2
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0.02, y: 4.5, z: 0}
+  m_LocalPosition: {x: -0.03, y: 4.69, z: 0}
   m_LocalScale: {x: 0.763, y: 0.6840596, z: 1}
   m_ConstrainProportionsScale: 0
   m_Children: []
@@ -2337,7 +2337,7 @@ Transform:
   m_GameObject: {fileID: 2141844658}
   serializedVersion: 2
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: -17.59, y: 3.26, z: -10}
+  m_LocalPosition: {x: -2.94, y: 2.98, z: -10}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_ConstrainProportionsScale: 0
   m_Children: []
-- 
2.51.0.windows.2

